src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1092,6 +1092,61 @@ def get_bucket_replication(self, bucket_name, request=None):
                     return None
             raise e
 
+    @RestClient.api_post('?Action=CreateTopic&Name={name}')
```

Comment:
always prefer lowercase for paths unless some external api/restriction.
```suggestion
    @RestClient.api_post('?action=createtopic&name={name}')
```

Reply:
"Suggested changes are done."

Reply:
@pujaoshahu @afreen23  https://docs.ceph.com/en/latest/radosgw/notifications/#create-a-topic.

I see the POST request documented as 
```
POST

Action=CreateTopic
&Name=<topic-name>
```
And I see CreateTopic as https://docs.aws.amazon.com/sns/latest/api/API_CreateTopic.html.

@pujaoshahu as @afreen23 mentioned here
> unless some external api/restriction.

this is an external endpoint that we are trying to consume and they are named `'?Action=CreateTopic&Name={name}')`. If the reviewer is unaware of it, please try to point them to the right documentation btw since most of us are new to the bucket notification.

Reply:
Okay, thanks for explanation, lets add a comment there just. 👍🏿